packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -3978,6 +4054,26 @@ Specifies a hashed password to import into Okta. This allows an existing passwor
 }
 ```
 
+##### Password Hook Object
+
+Specifies to use a password hook to import user into Okta. This allows an existing password to be imported into Okta directly from some other store. See [Create User with Password Hook](#create-user-with-password-hook) for information on using this object when creating a user. When updating a user with a password hook the user must be in the `STAGED` status.
```

Comment:
"Specifies to use a password hook to import user into Okta." > "Specifies that a [Password Import Inline Hook](/docs/reference/password-hook/) should be triggered to handle verification of the user's password the first time the user logs in."

Reply:
Done!